pkgs/tools/graphics/zbar/default.nix

```diff
@@ -17,6 +17,8 @@ in stdenv.mkDerivation rec {
     sha256 = "0hlxakpyjg4q9hp7yp3har1n78341b4knwyll28hn48vykg28pza";
   };
 
+  outputs = [ "out" "dev" ];
```

Comment:
we actually already went a lot further with this
* https://github.com/NixOS/nixpkgs/commit/e1cb839d17d7ec6bdf97624e0f1c0f0af6fb62c3#diff-b15562677b44058bddf61ce5bc1b5c20

There's other commits there that clean up dependencies etc. on master that perhaps we can backport.

Reply:
I wasn't aware of that pull request, thanks! I will backport that instead, as I prefer not to diverge from master.